### ISSUE TITLE
Python 3 support

### ISSUE THREAD

**Issue by Jenselme** [oberstet/taschenmesser#3]
Hi,

Can you add python 3 support? After reading the code, it seems that only some print statements must be modified. I should be able to provide the patch if you are interested.

Maybe also add a small test suite to check that everything works fine.

**Comment by oberstet** [comment]
yeah, both would be neat! I don't have time to work on this though =(
